docs/source/user-guide/installation.md

```diff
@@ -30,17 +30,22 @@ Make sure the following requirements are met and software installed on your mach
 
 ### Supported Spark Versions
 
-Comet currently supports the following versions of Apache Spark:
+### Supported Apache Spark Versions
+For more details, refer to the documentation: [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
+
+- **Apache Spark 3.3, 3.4, or 3.5**
+  - **Supported Java Versions**: JDK 8 and above
+  - **GLIBC**: 2.17 (CentOS 7) and above
+  - **Scala**: 2.12/2.13
+- **Apache Spark 4.0** *(Experimental Support - is intended for development/testing use only and should not be used in production yet.)*
+  - **Supported Java Versions**: JDK 17/21 
+  - **GLIBC**: 2.17 (CentOS 7) and above
```

Comment:
for GLIBC Im thinking if its good to specify a Linux distribution... I would leave just a version and put a command to check GLIBC like `ldd --version`